Taskfile.yml

```diff
@@ -237,32 +237,27 @@ tasks:
     cmds:
       - docker buildx create --driver=docker-container --name=ferretdb
 
-  docker-build:
-    deps: [build-testcover]
-    env:
-      CGO_ENABLED: 0
-      GOOS: linux
-    cmds:
-      - GOARCH=arm64 go test -c -o=bin/ferretdb-arm64 -trimpath -tags=testcover,tigris -coverpkg=./... ./cmd/ferretdb
-      - GOARCH=amd64 go test -c -o=bin/ferretdb-amd64 -trimpath -tags=testcover,tigris -coverpkg=./... ./cmd/ferretdb
-
   docker-local:
-    deps: [docker-build]
+    deps: [gen-version]
     cmds:
       - >
         docker buildx build --builder=ferretdb
+        --build-arg BUILD_ARGS="{{.BUILD_ARGS}}"
+        --build-arg CGO_ENABLED={{.CGO_ENABLED}}
```

Comment:
Why we only add them there, but not in `docker-push`?

Reply:
Those args have default values in [Dockerfile](https://github.com/FerretDB/FerretDB/pull/605/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R6-R7 ). Is it better to pass them explicitly?